dotnet/src/Planners/Planners.Core/IPlanner.cs

```diff
@@ -11,9 +11,9 @@ namespace Microsoft.SemanticKernel.Planners;
 #pragma warning restore IDE0130
 
 /// <summary>
-/// Interface for planner that uses a set of semantic functions to create a sequential plan.
+/// Interface for a planner that creates a plan to achieve a goal.
 /// </summary>
-public interface ISequentialPlanner
+public interface IPlanner
```

Comment:
Any reason not to move this to ms.sk.core? Or move both `IPlanner` and `Plan` to ms.sk.abstractions?